### PR TITLE
feat(react-recipes): Adding MediaObject recipe and Example template

### LINKED ISSUE #24765: Recipes: Write Media Object recipe
There could be two options, one could be to add a recipes section in our docs showing how to build a media object with current components and the other could be to add another sub-component (mayabe MediaObject)

### PR DESCRIPTION
**Pull request by sopranopillow**
This PR adds the following:
* MediaObject recipe.
* Example template similar to the v9-docsite's examples.

## Related Issue(s)

* Fixes #24765

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/f62eade5a07f637d4c460d4bc14b8ba473d37086'>f62eade5a07f637d4c460d4bc14b8ba473d37086</a></sub>
<div id="bundleSizeComment9423"/>

File: apps/recipes-react-components/src/recipes/media-object/MediaObject.stories.mdx
Comment (spmonahan) [inline apps/recipes-react-components/src/recipes/media-object/MediaObject.stories.mdx:221]: "dom" should be "DOM".

File: apps/recipes-react-components/src/recipes/media-object/MediaObject.stories.mdx
Comment (Hotell) [reply apps/recipes-react-components/src/recipes/media-object/MediaObject.stories.mdx:42]: From usage looks like it renders actual control

Comment (sopranopillow) [reply apps/recipes-react-components/src/recipes/media-object/MediaObject.stories.mdx:42]: correct, this renders a control

File: apps/recipes-react-components/src/recipes/media-object/MediaObject.stories.mdx
Comment (sopranopillow) [reply apps/recipes-react-components/src/recipes/media-object/MediaObject.stories.mdx:85]: This is to make the rendered example centered.

image

I can remove it thought it would be nice to add for future recipes since if it's not there, then we need to add another div to center the example.

**Comment by sopranopillow** [comment]
@Hotell I realize that maybe it's not convenient to have `@microsoft/fluentui-react-build` as codeowner, I can change it for `@microsoft/fluentui-react` if prefered. What do you think about it?

File: apps/recipes-react-components/src/recipes/media-object/MediaObject.stories.mdx
Comment (spmonahan) [reply apps/recipes-react-components/src/recipes/media-object/MediaObject.stories.mdx:88]: Ah, I see. It's not in the example guide but it is in the actual example code. My thinking is that I'd expect to be able to copy-paste any of the examples and have them work. I find it a little confusing that `centered` shows up in the example but the styles/props are not used anywhere in the guide.

**Comment by Hotell** [comment]
> @Hotell I realize that maybe it's not convenient to have `@microsoft/fluentui-react-build` as codeowner, I can change it for `@microsoft/fluentui-react` if prefered. What do you think about it?

sounds good! ty

**Review by Hotell — APPROVED** [review]
make sure to update your branch before merging ( we migrated to prettier 2.8 )

and run 
`yarn`
`yarn format --all`

submit any changes 

merge

ty

**Pull request merged**
